source/MRViewer/MRViewerSettingsManager.cpp

```diff
@@ -335,10 +335,12 @@ void ViewerSettingsManager::saveSettings( const Viewer& viewer )
 
     if ( ribbonMenu )
     {
-        auto& quickAccessList = ribbonMenu->getToolbar().getItemsList();
+        const auto& toolbar = ribbonMenu->getToolbar();
+        const auto& quickAccessList = toolbar.getItemsList();
         Json::Value qaList = Json::arrayValue;
-        qaList.resize( int( quickAccessList.size() ) );
-        for ( int i = 0; i < quickAccessList.size(); ++i )
+        qaList.resize( int( quickAccessList.size() ) + 1 );
+        qaList[0]["Version"] = toolbar.getItemsListVersion();
```

Comment:
better have JSON object with version and array than array with inconsistent types